src/ReportManager/ReportDetailView/index.js

```diff
@@ -727,6 +727,64 @@ const ReportDetailView = ({
 
   const isReadOnly = reportSchemas?.schema?.readonly;
 
+  const efb = {
+    'json': {
+      '$schema': 'https://json-schema.org/draft/2020-12/schema',
+      'additionalProperties': false,
+      'properties': {
+        'this_is_a_text': {
+          'default': '',
+          'deprecated': false,
```

Comment:
`deprecated: false` doesn't necessarily belong in a schema, let alone a fixture for testing. We only want this to validate that `deprecated: true` fields are hidden from display and rendering.